source/_integrations/zwave_js.markdown

```diff
@@ -1025,6 +1025,12 @@ The following table lists the Command Classes together with the implemented vers
 | Version                       | 3       | Highest granted |
 | Z-Wave Plus Info              | 2       | None            |
 
+{% note %}
+The documentation must list all cases where a "Working" of "Fail" status is returned for a valid and supported command of the Supervision Command Class:
```

Comment:
Is this intended to be kept? Sounds weird to talk about what the documentation must do.

Reply:
agreed, it is weird. I put it there because just the statement itself felt out of context.